runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -1055,16 +1055,8 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadBatchToSpaceND(const Operator
   auto block_shape = inputs.at(1);
   auto crops = inputs.at(2);
 
-  if (!subg.operands().at(crops).isConstant())
-    throw std::runtime_error("BatchToSpaceND: non-constant 'crops' is not supported.");
-
-  std::vector<std::int32_t> crops_v = subg.operands().at(crops).template asVector<std::int32_t>();
-  assert(crops_v.size() == 4);
-  if (crops_v != std::vector<std::int32_t>{0, 0, 0, 0})
-    throw std::runtime_error("BatchToSpaceND: 'crops' other than {0, 0, 0, 0} is not supported.");
-
```

Comment:
We should check this on acl-cl backend and acl-neon backend because they don't support non-zero crops.
Let's keep going on this PR. I'll make issue to resolve this later.